modules/javafx.graphics/src/main/java/javafx/scene/text/TabStopPolicy.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package javafx.scene.text;
+
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.layout.Region;
+
+/**
+ * The TabStopPolicy determines the tab stop positions within the text layout.
+ *
+ * @since 999 TODO
+ */
+public final class TabStopPolicy {
+    private final ObservableList<TabStop> tabStops = FXCollections.observableArrayList();
+    private final SimpleDoubleProperty defaultStops = new SimpleDoubleProperty(0.0);
+
+    /**
+     * Constructs a new {@code TabStopPolicy} instance.
+     */
+    public TabStopPolicy() {
+    }
+
+    /**
+     * The list of tab stops.
+     *
+     * @return the non-null, list of tab stops
+     */
+    public final ObservableList<TabStop> tabStops() {
+        return tabStops;
+    }
+
+    /**
+     * Provides default tab stops (beyond the last tab stop specified by {@code #tabStops()},
+     * as a fixed repeating distance in pixels from the last tab stop position.
+     * The position of default tab stops is computed at regular intervals relative to
+     * the leading edge of the {@code TextFlow} this policy is registered with.
+     * <p>
+     * The value of less than or equal 0 disables the default stops.
+     *
+     * @return the default tab stops property
+     * @defaultValue 0
```

Comment:
Would "8" be a better default value? I think having tabs do something (instead of effectively being ignored) by default when you create a TabStopPolicy would be less surprising to app developers.

Reply:
No, this value is in pixels (unlike the `tabSize` property).  I think it's better to let the application specify it explicitly.

Reply:
OK, this seems like a good reason to leave it.